### ISSUE TITLE
A11y_Visual Studio Code Client_Command Palette_Go to File_Color Contrast: Color contrast ratio of the "search files by name" placeholder text is "2.283:1" which is less than 4.5:1.

### ISSUE THREAD

**Issue by Rama8345** [microsoft/vscode#203656]
[[Check out Accessibility Insights!](https://accessibilityinsights.io/)]- Identify accessibility bugs before check-in and make bug fixing faster and easier.

### GitHubTags:
#A11yMAS; #A11yTCS; #A11ySev2; #DesktopApp; #Win32; #Visual Studio Code Client; #BM-VisualStudioCodeClient-Win32-Jan2024; #WCAG1.4.3; #Color Contrast;

### Environment and OS details:
Visual Studio Code insider version 1.86.0
Microsoft Windows 11 Enterprise 22H2 Build 22621.3007

### Repro Steps:

1. Open Visual Studio Code Insiders editor
2. Press ctrl+P or Tab till go and press ENTER then TAB till Go to File.
3. Observe that Color contrast ratio of the "search files by name" placeholder text is "2.283:1" which is less than 4.5:1.

### Actual:
Color contrast ratio of the "search files by name" placeholder text is "2.283:1" which is less than 4.5:1.

**Similar issue is also observed throughout the application.**

### Expected:
Color contrast ratio of the "search files by name" placeholder text should be greater than or equals to 4.5:1.

### User Impact:
Low vision users will not be able to see the placeholder text if color contrast ratio will be less than 4.5:1.

### Attachment:
![A11y_Visual Studio Code Client_Command Palette_Go to File_Color Contrast](https://github.com/microsoft/vscode/assets/115216656/767e5563-dafa-4539-964a-db78d5ed1bf5)

![Screenshot 2024-01-29 131417](https://github.com/microsoft/vscode/assets/115216656/f8315eac-a684-483e-9a61-2eae5d1029f2)

**Comment by hbons** [comment]
I can't reproduce this one. With the Dark Modern theme I see a background value for inputs of `#313131` which passes the 3:1 ratio with the placeholder foreground of `#818181`.

**Comment by hbons** [comment]
Misread that the ratio should be 4.5:1. This I can fix.

**Comment by kupatkar99** [comment]
Verified the issue in below environment:
Visual Studio Code Insider
Version: 1.87.0-insider (user setup)
Issue is fixed. Hence Closing the bug. please find added attachment for reference.
![Uploading 203656_Fixed.png…]()
